src/Components/CreateEventForm.tsx

```diff
@@ -16,20 +16,21 @@ const [volunteers, setVolunteers] = useState<CreatedEvent["vols_required"]>(0)
 const [description, setDescription] = useState<CreatedEvent['description']>('')
 const [eventName, setEventName] = useState<CreatedEvent['name']>('')
 const { org } = useContext(EventContext)
+console.log(org.phone)
 
 const submitEvent = (e:React.MouseEvent) => {
   e.preventDefault()
   const newEvent = {
     organization_id: org.id,
-    name: eventName,
-    phone: org.phone,
+    name: org.eventName,
     address: address,
+    phone: '(555)456-4564',
```

Comment:
oddest thing ever! Definitely a question for the backend. Maybe 555-555-5555 is some sort of reserved number?